flocker/node/agents/functional/test_gce.py

```diff
@@ -113,6 +111,40 @@ def test_attach_elsewhere_attached_volume(self):
         # the hack.
         pass
 
+    def test_create_volume_gold_profile(self):
```

Comment:
Add some short docstrings for these new tests. 
Or replace them with `make_iprofile_blockdevice_tests` if possible.